keras/testing_infra/test_combinations.py

```diff
@@ -497,12 +497,13 @@ def keras_mode_combinations(mode=None, run_eagerly=None):
 
     Args:
       mode: List of modes to run the tests. The valid options are 'graph' and
-        'eager'. Default to ['graph', 'eager'] if not specified. If a empty list
-        is provide, then the test will run under the context based on tf's
-        version, eg graph for v1 and eager for v2.
+        'eager'. If None, uses ['graph', 'eager']. If an empty
+        list is provided, then the test will run under the context based on
+        tensorflow's version, e.g., graph for v1 and eager for v2. Defaults to
+        `None`.
       run_eagerly: List of `run_eagerly` value to be run with the tests.
-        Default to [True, False] if not specified. Note that for `graph` mode,
-        run_eagerly value will only be False.
+        When None, uses [True, False]. Note that for `graph` mode,
```

Comment:
?

Reply:
@fchollet `Defaults to` has a particular syntax in the docstring format. I want to treat your codebase—and other Python codebases—as a data schema, one that can be translated into SQL, CLI, and others.

Unexpected other advantages is being able to automatically add type annotations, or push them back to the docstring.

In this particular example, the argument does not default to `[True, False] if not specified`. It could default to `[True, False]` but the implementation interface doesn't indicate this.